test/person.py

```diff
@@ -0,0 +1,12 @@
+from ies_tool.ies_tool import IESTool, Person
```

Comment:
I'm just thinking about these sorts of file, and whether they are really useful. A test is only really a test when we know what the success is. Would it be better to make this in to a unittest that checks the correct thing is written? Or should there at least be a comment showing what output is expected in the ttl file?

If the file is only intended for local development then it doesn't really need to be included at all.

Reply:
Yes - agreed - these need to be turned into formal tests at some point.